ecmd-core/ext/template/cmd/makefile.cmd

```diff
@@ -48,6 +49,9 @@ install:
 	@cp ${EXTENSION_NAME}Interpreter.H ${INSTALL_PATH}/ext/${EXTENSION_NAME}/cmd/.
 	@cp ../capi/${EXTENSION_NAME}Structs.H ${INSTALL_PATH}/ext/${EXTENSION_NAME}/cmd/.
 	@cp ../capi/${EXTENSION_NAME}ClientCapi.H ${INSTALL_PATH}/ext/${EXTENSION_NAME}/cmd/.
+	@echo "Installing Autogenerated header files to ${INSTALL_PATH}/extra/ ..."
+	@mkdir -p ${INSTALL_PATH}/extra/
+	cp ${EXTRA_INCLUDES} ${INSTALL_PATH}/extra/
```

Comment:
IMO, if these headers are needed by clients there really isn't any point in differentiating them.  I would just put them in capi with all the rest.  Lets see what Jason thinks though.

Reply:
Are all the headers from the SRCPATH needed? or is it just *DllCapi.H files?
If so a line could be added for those here.